### DISCUSSION TITLE
Getting issue of whitelist URI

### DISCUSSION THREAD

**Discussion by Umairhassan5** [Web3Auth/Web3Auth#1132] [Help]
Hi, i am facing the issue currently of whitelisted URl but i have created new project and gave the ID as well adding new RPC Matic test URl. Will you please check why i am getting this issue of URL whitelisted(url). Please check the attach images . I have changed the network with adaptor library but facing same issue again and again. 
<img width="1438" alt="screenshot" src="https://user-images.githubusercontent.com/97150180/209546793-09a6d594-d046-466d-b848-cce91d79e30b.png">

<img width="939" alt="code" src="https://user-images.githubusercontent.com/97150180/209546800-558bc937-d50e-4afa-8075-9aabd35b5a70.png">

**Comment by shahbaz17** [comment]
Hey @Umairhassan5 

Have you whitelisted https://dev.racino.io on the Web3Auth developer dashboard?

https://web3auth.io/docs/developer-dashboard/get-client-id#add-a-new-whitelist-url

**Reply by Umairhassan5** [reply]
Yes i have whitelisted the URL.

**Reply by shahbaz17** [reply]
Can you share the code snippet?
